README.md

```diff
@@ -78,9 +78,13 @@ server that is viewable with Kibana and Grafana!
 
 Look at some of the other benchmarks for examples of how this works.
 
+Note: Please tag @aakarshg for review when adding a new benchmark wrapper especially ones that index data into elasticsearch and mention:
```

Comment:
ok for now but long-term we can't have you responsible for every single benchmark getting a mapping in every single ES server.   Is there a way you can train the rest of us to help do this?

Reply:
Agreed. This isn't maintainable.

Reply:
Isn't there a way in Kibana to define this?  Or does that only apply to using Kibana?